storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -14,9 +14,9 @@ include::modules/technology-preview.adoc[leveloffset=+1]
 
 Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
 
-To create CSI-provisioned PVs that mount to AWS EBS storage assets, {product-title} installs the AWS EBS CSI Driver Operator and the AWS EBS CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+To create CSI-provisioned PVs that mount to AWS EBS storage assets, you must install the AWS EBS CSI Driver Operator and the AWS EBS CSI driver in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
If this is correct, then how does the user install the Operator, through OperatorHub?